lisa/schema.py

```diff
@@ -381,6 +381,19 @@ class NodeSpace(search_space.RequirementMixin, TypedSchema, ExtendableSchemaMixi
         default=search_space.IntRange(min=0),
         metadata=metadata(decoder=search_space.decode_count_space),
     )
+    data_disk_caching: str = field(
```

Comment:
`data_disk_caching_type`

Reply:
Move it near to `data_disk_count`. The similar things should be together.

Reply:
fix